examples/comment_defaults.py

```
@@ -0,0 +1,106 @@
+    name: str = "my-server"
+    debug: bool = False
+    database: DatabaseConfig = field(default_factory=DatabaseConfig)
+    logging: LoggingConfig = field(default_factory=LoggingConfig)
+
+
+if __name__ == "__main__":
+    cfg = ServerConfig(
+        database=DatabaseConfig(port=3306)
+    )  # override default port to show how it appears in output files
+
+    toml_path = Path(tempfile.mktemp(suffix=".toml"))
```

Ruff (0.15.10)

[error] Use of insecure and deprecated function (`mktemp`)

(S306)